Automattic/wp-calypso
# Update nav in Jetpack cloud pricing page

### Changes proposed in this Pull Request

This PR updates the navigation in the Jetpack cloud pricing page to reflect the new Jetpack.com nav.

### Testing instructions

- Download this PR and run Jetpack cloud
- Visit `/pricing`
- Check that the navigation (except the _Get started_ and search buttons) looks and behaves like the navigation in https://jetpack.com/ (it shouldn't stick as you scroll, though)
- Check that nav items and sub items dispatched the tracking event `calypso_jetpack_nav_item_click` when clicked, with the proper `nav_item` parameter

### Screenshots

_Desktop_
<img width="1002" alt="Screen Shot 2021-07-16 at 3 04 30 PM" src="https://user-images.githubusercontent.com/1620183/125999209-0ac64319-60eb-45da-82c9-3fe06c80378c.png">


_Mobile_
<img width="378" alt="Screen Shot 2021-07-16 at 3 28 23 PM" src="https://user-images.githubusercontent.com/1620183/125999193-97b86be8-7440-47bb-b0bc-2af570ce06ad.png">
